misk-admin/src/test/kotlin/misk/web/metadata/MiskWebFormBuilderTest.kt

```diff
@@ -50,6 +54,13 @@ internal class MiskWebFormBuilderTest {
       )
     )
 
+    // Check map value types are included
```

Comment:
Could you add assertions that the key type (in this case int32) is also present in the calculated types?